model/map-jpa/pom.xml

```diff
@@ -51,7 +51,7 @@
             <plugin>
                 <groupId>org.hibernate.orm.tooling</groupId>
                 <artifactId>hibernate-enhance-maven-plugin</artifactId>
-                <version>${hibernate-orm.version}</version>
+                <version>6.2.0.Final</version>
```

Comment:
I believe we need a review from @keycloak/store.

Reply:
I would say it makes sense to have the property defined in the parent pom and then use it here.

Reply:
Ok, updated in latest push